cohere/client.py

```diff
@@ -1047,7 +1047,7 @@ def wait_for_cluster_job(
 
     def create_embed_job(
         self,
-        input_dataset: Union[str, BaseDataset],
+        dataset: Union[str, BaseDataset],
```

Comment:
Billy wants to remove the union type, keep the name consistent between api and sdk. Would be good to get his input

Reply:
https://github.com/cohere-ai/blobheart/pull/14120#discussion_r1451608283